client/src/components/Error404.js

```diff
@@ -1,13 +1,31 @@
 import React, { Fragment } from 'react';
+import ReactRouterPropTypes from 'react-router-prop-types';
 import CHeader from './CHeader';
+import * as S from './Error404.style';
+import Button from './CBtn';
 
-const Error404 = () => {
+const Error404 = props => {
+  const back = () => {
+    const { history } = props;
+    history.go(-1);
+  };
   return (
     <Fragment>
       <CHeader menu={null} />
-      <h1>Error404</h1>
+      <S.Main>
+        <S.Div>
+          <S.H1>
+            404!
+            <br />
+            Sorry, the page you are looking for is unavailable.
```

Comment:
I think this would be better broken up into an h1 and an h2.

E.g.
```
<h1>404!</h1>
<h2>Sorry, the page you are looking for is unavailable.</h2>
```